clang/include/clang/CConv/ConstraintsGraph.h

```diff
@@ -161,8 +164,17 @@ class DataGraph :
     auto *N = this->findNode(NodeType(Start));
     if (N == this->end())
       return;
-    for (auto TNode : llvm::breadth_first(*N))
-      Fn(TNode->getData());
+    // Insert into BFS cache.
+    if (BFSCache.find(Start) == BFSCache.end()) {
+      std::set<Data> ReachableNodes;
+      ReachableNodes.clear();
```

Comment:
Why do you need to call `clear()` here? Isn't just creating the variable calling its constructor, which will ensure that it's empty?

Reply:
Agree. Will remove it.